src/net/sparkzz/modest/utils/Logger.java

```diff
@@ -2,10 +2,7 @@
 
 import net.sparkzz.modest.io.FileManager;
 
-import java.io.File;
```

Comment:
Unused imports. Just a waste of memory :) If you need them later, you can always add them again.